lib/data_update_scripts/20210210124346_add_data_updates_scripts_feature_flag.rb

```diff
@@ -0,0 +1,7 @@
+module DataUpdateScripts
+  class AddDataUpdatesScriptsFeatureFlag
+    def run
+      FeatureFlag.add(:data_update_scripts)
```

Comment:
Rather than run this script on every Forem, would it be easier to just add this via admin to DEV through the UI and then not worry about creating this script?

Reply:
I'm the one that suggested adding the script as I've done it at least once in the past and added it again to a recent PR - #12627 - by adding it manually it means that going to a Forem that doesn't have this enabled means that the operator won't know which feature flags are available.

As there's a difference between "available" and "enabled".

This way we can also use a script to remove it when the temporary feature is finalized, making sure we don't leave feature flags around in the various Forems.

Reply:
For example, right now on DEV there's a feature flag I'm not sure if it's still used or not. To know that I'd have to grep the code and then remove it. I think by doing this explicitly and automatically we can avoid this situation.